src/Common/src/System/Net/Security/CertificateHelper.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
If it can be merged in the future with S.N.Security then it's OK to leave it here. If there are no plans for that, it should probably be moved to a \Http folder to avoid a confusion.